### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by ThomasLeigh**

### CODE REVIEW

File: FlymFork/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -27,31 +27,31 @@
     <string name="about_flym">About the app / Help / Contact developer. </string>
     <string name="welcome_title">Welcome to Flym Fork. Would you like to add a feed?</string>
     <string name="never">never</string>
-    <string name="update">Update</string>
+    <string name="update">update</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values/strings.xml]: This is a caption

File: FlymFork/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -27,31 +27,31 @@
...
     <string name="colon">:\u00A0</string>
     <string name="menu_settings">Settings</string>
     <string name="menu_refresh">Refresh</string>
     <string name="menu_all_read">Mark all as read</string>
-    <string name="flym_feeds">Flym feeds</string>
+    <string name="flym_feeds">Flym Feeds</string>
     <string name="error">Error</string>
-    <string name="network_error">The network is not set up</string>
+    <string name="network_error">The network is not (properly) set up</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values/strings.xml]: This is not manual setup. It if the state of radio module

File: FlymFork/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -120,12 +120,12 @@
...
-    <string name="menu_reload_full_text_without_mobilizer">Reload without mobilizer</string>
-    <string name="share_favorites_title">Flym: starred entries</string>
+    <string name="menu_reload_full_text_without_mobilizer">Reload without Mobilizer</string>
+    <string name="share_favorites_title">Flym Fork: starred entries</string>
     <string name="copied_clipboard">Link copied</string>
-    <string name="get_full_text">Load article full text</string>
-    <string name="show_text_in_entry_list">Show article text in list</string>
-    <string name="original_text">Show original text</string>
+    <string name="get_full_text">Load full article.</string>
+    <string name="show_text_in_entry_list">Show excerpts on article list.</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values/strings.xml]: This is not excerpts, it is article rss full text

File: FlymFork/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -226,11 +226,11 @@
     <string name="fontsize_entrylist">Font size in articles'\ list</string>
     <string name="settings_volume_buttons_action">Action by volume buttons</string>
     <string name="behaviour">Behavior</string>
-    <string name="question_delete_entry">Delete article ?</string>
+    <string name="question_delete_entry">Delete this article?</string>
     <string name="share_all_text">Share whole text</string>
     <string name="group">Group</string>
     <string name="auto_image_load">Auto images load</string>
-    <string name="show_progress_info">Detailed progress info</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values/strings.xml]: Is PROGRESS technical detail info

**Comment by yanus171** [comment]
Thank you Thomas. I didnt mind that there were SO many missspellings. O my poor English !!! But I ask you to revert back some strings that I commented, after that I can apply your changes

File: FlymFork/src/main/res/values/strings.xml (hunk 3 above)
Comment (yanus171) [reply FlymFork/src/main/res/values/strings.xml]: By the way, yesterday one of my Russian users asked me to add a NEW feature to show serveral first lines of article text in article list - for this word excerpts sounds much better.

Comment (yanus171) [reply FlymFork/src/main/res/values/strings.xml]: Livejournal rss channel contains FULL text of article. And so some others. So It is not as easy to decide. Lets leave it ORIGINAL :) for now

**Comment by yanus171** [comment]
>> Updating feeds... please wait.
User do NOT needs to wait while feed is updating - it can see other feeds or articles

File: FlymFork/src/main/res/values/strings.xml (hunk 1 above)
Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: I've checked where it is displayed and it is one of the verbs which inform on what is happening (on update) - those verbs which we've discussed earlier which should have the "-ing" ending. So I'll change it that way ("Updating...").

**Comment by yanus171** [comment]
Ok, but without ... Did you see all my comments on github page?


>Четверг,  5 июля 2018, 21:12 +03:00 от Thomas Leigh <notifications@github.com>:
>
>@ThomasLeigh commented on this pull request.
>----------------------------------------------------------------------
>In  FlymFork/src/main/res/values/strings.xml :
>> @@ -27,31 +27,31 @@
>     <string name="about_flym">About the app / Help / Contact developer. </string>
>     <string name="welcome_title">Welcome to Flym Fork. Would you like to add a feed?</string>
>     <string name="never">never</string>
>-    <string name="update">Update</string>
>+    <string name="update">update</string>
>
>I've checked where it is displayed and it is one of the verbs which inform on what is happening (on update) - those verbs which we've discussed earlier which should have the "-ing" ending. So I'll change it that way ("Updating...").
>—
>You are receiving this because you commented.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением, Алексей

File: FlymFork/src/main/res/values/strings.xml (hunk 4 above)
Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: I propose "Detailed progress technical info".

**Comment by yanus171** [comment]
ok


>Четверг,  5 июля 2018, 21:17 +03:00 от Thomas Leigh <notifications@github.com>:
>
>@ThomasLeigh commented on this pull request.
>----------------------------------------------------------------------
>In  FlymFork/src/main/res/values/strings.xml :
>>      <string name="share_all_text">Share whole text</string>
>     <string name="group">Group</string>
>     <string name="auto_image_load">Auto images load</string>
>-    <string name="show_progress_info">Detailed progress info</string>
>
>I propose "Detailed progress technical info".
>—
>You are receiving this because you commented.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением, Алексей

File: FlymFork/src/main/res/values/strings.xml (hunk 1 above)
Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: Should I change anything and again make pull requests - or just to comment here?

File: FlymFork/src/main/res/values/strings.xml (hunk 2 above)
Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: Ok.

File: FlymFork/src/main/res/values/strings.xml (hunk 3 above)
Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: Ok, let's leave "Show original text on article list.".

File: FlymFork/src/main/res/values/strings.xml (hunk 4 above)
Comment (yanus171) [reply FlymFork/src/main/res/values/strings.xml]: ok

**Pull request closed without merging**